reqMgrClient.py

```diff
@@ -399,8 +427,8 @@ def getWorkflowInfo(url, workflow):
     """
```

Comment:
Thanks Julian, the script you point is exactly what I wanted.
One more question, when you run this test do you set the environment using following script?

https://github.com/CMSCompOps/WmAgentScripts/blob/master/setenvscripts.sh

If not, how do you set the environment, more specifically which python version are you using for the test?